docs/plan-for-production.md

```diff
@@ -60,16 +60,16 @@ the different components of a Swam cluster listen on. You should use these to
 configure your firewalls and other network access control lists.
 
 - **Swarm manager.**
-    - **Inbound 80/tcp (HTTP)**. This is allows `docker pull` commands to work. If you will be pulling from Docker Hub you will need to allow connections on port 80 from the internet.
+    - **Inbound 80/tcp (HTTP)**. This allows `docker pull` commands to work. If you will be pulling from Docker Hub you will need to allow connections on port 80 from the internet.

```

Comment:
Change 

```
If you will be pulling from Docker Hub you will need to allow connections on port 80 from the internet
``

to

```
If you plan to pull from Docker Hub, you must allow Internet connections to port 80.
```